plaso/helpers/macos/location.yaml

```diff
@@ -0,0 +1,145 @@
+# dtFabric format specification.
+---
+name: aul_location
+type: format
+description: Apple Unified Logging and Activity Tracing formats
+urls: ["https://github.com/libyal/dtformats/blob/main/documentation/Apple%20Unified%20Logging%20and%20Activity%20Tracing%20formats.ascidoc"]
+---
+name: byte
+type: integer
+attributes:
+  format: unsigned
+  size: 1
+  units: bytes
+---
+name: bool8
+type: boolean
+attributes:
+  size: 1
+  units: bytes
+  false_value: false
```

Comment:
This is not correct use of this see: https://dtfabric.readthedocs.io/en/latest/sources/Format-specification.html#boolean

what should this be? false if 0, true otherwise?